### PR TITLE
Feature/new projects and activities badges

### PR DESCRIPTION
**Pull request by o-mehlii**

### CODE REVIEW

File: renderer/components/ui/AutocompleteSelector.tsx [hunk 1]
```diff
@@ -94,6 +98,9 @@ export default function AutocompleteSelector({
     if (selectedItem.startsWith("TT:: ")) {
       setSelectedItem((prev) => prev.slice(5));
     }
+    if (isNewCheck) {
+      setIsNew(selectedItem && !availableItems.includes(selectedItem));
+    }
```
Comment (d-mashchenko) [inline renderer/components/ui/AutocompleteSelector.tsx:103]: we need to make sure availableItems will never be undefined or use optiaonal chaining

**Comment by d-mashchenko** [comment]
In general: do we need to check each change that project is new? I think it's enough just add this note after blur

**Review by a-nahaitsev — CHANGES_REQUESTED** [review]
Agree with Dima's comment.
And it would be better to show Badge when onBlur

**Review by d-mashchenko — APPROVED** [review]

File: renderer/components/ui/AutocompleteSelector.tsx [hunk 2]
```diff
@@ -121,6 +134,11 @@ export default function AutocompleteSelector({
           )}
           onChange={onChangeHandler}
           tabIndex={tabIndex}
+          onBlur={() => {
+            if (isNewCheck && availableItems) {
+              setIsNew(selectedItem && !availableItems.includes(selectedItem));
+            }
+          }}
```
Comment (a-nahaitsev) [inline renderer/components/ui/AutocompleteSelector.tsx:141]: move it to handler function

**Review by a-nahaitsev — CHANGES_REQUESTED** [review]

**Pull request merged**